torch/_torch_docs.py

```diff
@@ -4530,6 +4530,11 @@ def merge_dicts(*dicts):
 
 Returns the median value of all elements in the :attr:`input` tensor.
 
+.. note::
+    For even-sized inputs, the smaller of the two middle elements in the sorted
```

Comment:
"even-sized inputs" is a problem. In the first definition of median the tensor is flatted, so it's inputs with an even number of elements. In the second definition of median the median is computed along its a dimension, so it's whether that dimension has an even size.